sections/introduction.tex

```diff
@@ -2,18 +2,18 @@ \chapter{Introduction} % (fold)
 \label{cha:introduction}
 Ever since the dawn of mathematics, mathematicians has strived for faster computations while maintaining correctness.
 In our time computers have become the main target for doing extensive computations, and their CPUs are what the majority of programming languages apply to.
-CPUs are good at handling sequential instructions and moderns CPUs even exploit their multiple cores in order to attend to serial and task parallel workloads.
+CPUs are good at handling sequential instructions and modern CPUs even exploit their multiple cores in order to attend to serial and task parallel workloads.
```

Comment:
Her kan man måske også skrive noget om at programmeringssprog er blevet lavet som gør det muligt at udnytte denne type CPU, for også at dreje introduktionen lidt ind på sprog fremfor det bare er hardware :) ?